MUDscape/commands.py

```diff
@@ -16,19 +16,19 @@ def help(cls):
 
 
 class Look(Command):
-    tokens = ['l', 'map', 'm']
+    tokens = ['look', 'l', 'map', 'm']
 
     @staticmethod
     def perform(session):
         # TODO: Decide on map storage method
-        return """\
+        return ("""\
 ╔═══════════════╗
 ║               ║
 ║               ║
 ║       X       ║
 ║               ║
 ║               ║
-╚═══════════════╝"""
+╚═══════════════╝""").encode('utf-8')
```

Comment:
forgot to delete that after testing to see If I could get it to print properly on windows; right now it's printing garbage characters.